phobos/smurf/robot.py

```diff
@@ -6,20 +6,20 @@
 import numpy as np
 import yaml
 
-from ..core import Robot
+from ..core import Robot, derive_model_dictionary
 from .motors import Motor
 from .poses import Pose
 from .core import Material, Collision, Joint, Link
 from .hyrodyn import Submechanism, Exoskeleton
-from .sensors import Sensor, MultiSensor
+from .sensors import *
```

Comment:
Please add only those you need, importing everything might be dangerous when leading to shadowing. See smurf/__init__.py how the sensors are imported there